core/controllers/base.py

```diff
@@ -168,10 +168,19 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
         # Initializes the return dict for the handlers.
         self.values = {}
 
+        # This try-catch block is intended to log cases where getting the
+        # request payload errors with ValueError: Invalid boundary in multipart
+        # form: b''. This is done to gather sufficient data to help debug the
+        # error if it arises in the future.
+        try:
+            payload_json_string = self.request.get('payload')
```

Comment:
Wouldn't base_test.py need to be updated since the code here is being changed?

Reply:
Ah, yes that's why the tests were failing :D
Done!